src/components/Cards.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
I'm not a fan of having a collection component with a name like "Cards" If this is necessary I think you need to come up with a better name.

I think what this is trying to do is place components next to each other until the screen is too small.

What about a comopnent called Grid that does this for you.

The markdown might look like this.

```
<Grid md-cols="2">
  <Card>
  ...
  </Card>
  <Card>
  ...
  </Card>
</Grid>
```

Then the vue component looks like
```
<template>
 <div class="grid gap-6 lg:gap-12", :class="(something computed from properties)">
  <slot />
 </div>
</template>
```

Reply:
I was trying to reduce styling in the markdown file. I think changing the name is ok but I don't wholly agree with your suggestion. I will have a think about it. Also you can't pass more than one think into the slot

Reply:
My Suggestion wouldn't include any styling in the markdown file

Reply:
I have updated to your suggestions I haven't added the properties if someone wants to extend later they can